MathlibTest/Lint.lean

```diff
@@ -7,23 +7,6 @@ import Mathlib.Order.SetNotation
 
 set_option linter.dupNamespace false
 
-namespace termG
```

Comment:
I've removed this (now broken) test on @adomani's [advice](https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/Making.20Mathlib.20tests.20into.20a.20library/near/479175338).